src/html/fetch-markdown.js

```diff
@@ -70,51 +160,11 @@ async function fetch(context, { secrets = {}, request, logger }) {
   }
   if (!ref) {
     logger.warn(`Recoverable error: no ref given for ${repo}/${owner}.git${path}, falling back to master`);
-    ref = 'master';
   }
 
-  const { REPO_RAW_ROOT: rootPath } = secrets;
-
-  // everything looks good, make the HTTP request
-  const options = {
-    uri: uri(rootPath, owner, repo, ref, path),
-    json: false,
-    timeout,
-    time: true,
-  };
-
-  // if there is a github token, send it in the Authorization header
-  const token = secrets.GITHUB_TOKEN || (request.headers ? request.headers['x-github-token'] : '');
-  if (token) {
-    setdefault(options, 'headers', {
-      Authorization: `token ${token}`,
-    });
-  }
-
-  logger.debug(`fetching Markdown from ${options.uri}`);
-  try {
-    content.body = await client(options);
-    if (branch && branch !== ref) {
-      setdefault(content, 'sources', []).push(uri(rootPath, owner, repo, branch, path));
-    } else {
-      setdefault(content, 'sources', []).push(options.uri);
-    }
-  } catch (e) {
-    if (e.statusCode === 404) {
-      logger.error(`Could not find Markdown at ${options.uri}`);
-      setdefault(context, 'response', {}).status = 404;
-    } else if ((e.response && e.response.elapsedTime && e.response.elapsedTime > timeout) || (e.cause && e.cause.code && (e.cause.code === 'ESOCKETTIMEDOUT' || e.cause.code === 'ETIMEDOUT'))) {
-      // return gateway timeout
-      logger.error(`Gateway timeout of ${timeout} milliseconds exceeded for ${options.uri}`);
-      setdefault(context, 'response', {}).status = 504;
-    } else {
-      logger.error(`Error while fetching Markdown from ${options.uri} with the following `
-                   + `options:\n${inspect(options, { depth: null })}`);
-      setdefault(context, 'response', {}).status = 502;
-    }
-    context.error = e;
-  }
+  return await fetchMarkdown(context, action)
+    + await fetchMarkupConfig(context, action);
```

Comment:
is this `+` intentional ? looks a bit ugly... 

however, I would use `Promise.all()` so we can fetch the concurrently.